encrypted-media-respec.html

```diff
@@ -2326,7 +2325,7 @@
           <ol>
             <li><p>Let the <var>session</var> be the associated <a>MediaKeySession</a> object.</p></li>
             <li><p>Let <var>expiration time</var> be <code>NaN</code>.</p></li>
-            <li><p>If the new expiration time is not <code>NaN</code>, let <var>expiration time</var> be the new expiration time in milliseconds since 01 January 1970 UTC.</p></li>
+            <li><p>If the new expiration time is not <code>NaN</code>, let that be a new <var>expiration time</var>.</p></li>
```

Comment:
That does seem clearer.  I'll update to use "let <var>expiration time</var> be that expiration time" in both places.